### PR TITLE
fix: make schema singleton

### PR DESCRIPTION
**Pull request by jjtang1985**
The `get schema()` function of the api classes creates an object all the time, causing high memory usage.
This PR applies singleton pattern for this function.

- [x] generate all services
- [x] changelog update after release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 ## Improvements
 
--
+- [generator] Improve `get schema()` of API classes (e.g., `BusinessPartnerAPI`) from generated clients by applying singleton pattern, which reduces memory usage. It typically matters, when the function is called in a loop.
```
Comment (marikaner) [inline CHANGELOG.md:25]:
```suggestion
- [generator] Reduce memory usage of `schema` in API classes (e.g., `BusinessPartnerAPI`).
```

**Review by marikaner — APPROVED** [review]
I hope that solves most of the issues, did you check that the types work well for navigation properties?

**Comment by jjtang1985** [comment]
> I hope that solves most of the issues, did you check that the types work well for navigation properties?

Yes, for example, you can build filters like below with autocompletion:
```
commonEntityApi.schema.TO_SINGLE_LINK.filter()
```

**Comment by jjtang1985** [comment]
> I hope that solves most of the issues

I'll test it with canary version after merging.

**Pull request merged**